Foundation/Monad.hs

```diff
@@ -1,7 +1,6 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE DeriveGeneric #-}
 {-# LANGUAGE DeriveDataTypeable #-}
-{-# LANGUAGE StandaloneDeriving #-}
```

Comment:
I think this one is needed for older compilers (ghc.7.8 and 7.10).

see: https://travis-ci.org/haskell-foundation/foundation/jobs/241461362#L300